test/functional/ex_cmds/print_commands_spec.lua

```diff
@@ -0,0 +1,11 @@
+local helpers = require('test.functional.helpers')(after_each)
+local eq, command, funcs = helpers.eq, helpers.command, helpers.funcs
+
+describe(':z^', function()
+  before_each(clear)
+
+  it('correctly sets the cursor after :z^', function()
+    command('silent! z^')
```

Comment:
I initially added it because before the change, `z^` _did_ indirectly result in an error when I tried running it in a normal nvim instance (generated by the highlighting functionality). Anyway, you're absolutely right, I'll remove it.